src/main/Port.Adapter/IO/Persistence/SQLite/SubscriptionRepository.cs

```diff
@@ -46,7 +46,7 @@ public async Task<IList<Subscription>> GetAllByAvatarUrlSnapshotIdAsync(Guid ava
             return subscriptions.Select(s => new Subscription()
             {
                 Id = s.Id,
-                UserId = s.UserId,
+                UserNeuronId = s.UserId,
```

Comment:
Please rename s.UserId to s.UserNeuronId for consistency.

Reply:
I renamed the references in the `Persistence` project, but will require [a separate PR](https://github.com/ei8/avatar-server-pack/pull/9) to apply the renames to the SQLite database itself.